src/Database/Commands/QueryCommand.ts

```diff
@@ -47,18 +49,26 @@ export class QueryCommand extends RavenCommand {
     const query = this.indexQuery;
 
     this.params = {
-      waitForNonStaleResultsAsOfNow: 'false',
-      pageSize: query.pageSize,
-      start: query.start
+      waitForNonStaleResultsAsOfNow: 'true',
```

Comment:
Should be PascalCased: `WaitForNonStaleResultsAsOfNow`.
Also why to we need `params` and `payload` filled in? I assume it's work in progress - transition to using POST?